scripts/build.sh

```diff
@@ -23,9 +23,9 @@ mkdir dist/npm/syntaxes
 cp syntaxes/!(tsconfig).json dist/npm/syntaxes
 
 pushd dist/npm
-yarn install --production --ignore-scripts
```

Comment:
Would it be useful to add a TODO comment here that either links to the `vsce` issue or this PR so we know to revert this when the issue is fixed?

Reply:
Good call! Added TODO